ztp/source-crs/ClusterVersion.yaml

```diff
@@ -5,9 +5,10 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "100"
 spec:
-  # The default channel is stable-<version>
-  channel: $channel
-  # The default upstream path is https://api.openshift.com/api/upgrades_info/v1/graph
-  upstream: $upstream
-  # desiredUpdate:
-  #   version: $version
+  desiredUpdate:
+    force: false
+    version: $version
+status:
```

Comment:
I am afraid of precaching feature needs to look at the "upstream" from this policy. If that's the case, "upstream:https://api.openshift.com/api/upgrades_info/v1/graph" should be added in default source CR.
Probably @vitus133 can confirm

Reply:
Yes, this is a very valid concern. if you specify `version` you also need `channel` and `upstream`. All three are needed to get the release image pull spec.